src/__tests__/Form.feature

```diff
@@ -24,11 +24,11 @@ Feature: Form
     Then the "<fieldLabel>" field have the "<errorMessages>" error messages
 
     Examples:
-      | fieldLabel    | value | errorMessages                          |
-      | Firstname     |       | Field cannot be empty                  |
-      | Lastname      |       | Field cannot be empty                  |
-      | Email         |       | Field cannot be empty - Invalid format |
```

Comment:
tu supprimes le fait de vérifier le format de l'email ?

Reply:
Avec yup si le problème vient du format tu auras uniquement le message _Invalid Format_, si le champ est vide tu auras uniquement _Field cannot be empty_